src/analysis/souffle/examples/sql_examples/query6.dl

```diff
@@ -0,0 +1,29 @@
+#include "taint.dl"
+#include "sql_aux.dl"
+#include "policy.dl"
+
+isAccessPath("top.exprResult").
+edge("top.select_source.exprResult", "top.exprResult").
+isAccessPath("top.select_source.exprResult").
+edge("top.select_source.select_core.MULTIPLY.exprResult", "top.select_source.exprResult").
+isAccessPath("top.select_source.select_core.MULTIPLY.exprResult").
+edge("top.select_source.select_core.MULTIPLY.DIVIDE.exprResult", "top.select_source.select_core.MULTIPLY.exprResult").
+edge("literal_numeric_3600000", "top.select_source.select_core.MULTIPLY.exprResult").
+doesXform("top.select_source.select_core.MULTIPLY.exprResult", "MULTIPLY").
+isAccessPath("top.select_source.select_core.MULTIPLY.DIVIDE.exprResult").
+edge("top.select_source.select_core.MULTIPLY.DIVIDE.MULTIPLY.exprResult", "top.select_source.select_core.MULTIPLY.DIVIDE.exprResult").
+edge("literal_numeric_3600000", "top.select_source.select_core.MULTIPLY.DIVIDE.exprResult").
+doesXform("top.select_source.select_core.MULTIPLY.DIVIDE.exprResult", "DIVIDE").
+isAccessPath("top.select_source.select_core.MULTIPLY.DIVIDE.MULTIPLY.exprResult").
+edge("top.select_source.select_core.MULTIPLY.DIVIDE.MULTIPLY.birthTimestamp", "top.select_source.select_core.MULTIPLY.DIVIDE.MULTIPLY.exprResult").
+edge("literal_numeric_1000", "top.select_source.select_core.MULTIPLY.DIVIDE.MULTIPLY.exprResult").
+doesXform("top.select_source.select_core.MULTIPLY.DIVIDE.MULTIPLY.exprResult", "MULTIPLY").
+isAccessPath("top.select_source.select_core.MULTIPLY.DIVIDE.MULTIPLY.birthTimestamp").
```

Comment:
Do we really need to capture `MULTIPLY.DIVIDE.MULTIPLY` or is it sufficient to capture the fact that `birthTimestamp` is accessed?

Reply:
the `MULTIPLY.DIVIDE.MULTIPLY` is context as to where this occurs, it isn't attempting to capture the actual operations per se. That said, I have realized reading this column that if you performed the same calculation in two different expressions in a `SELECT` list, they would have conflicting access paths. I will fix that issue.